features/SideNavigation.feature

```diff
@@ -0,0 +1,165 @@
+@owner:lilla
+@author:lilla
+
+Feature: Navigating Sidenav
+    This allows users to easily navigate and select between articles and categories within our help center site
+
+    @ui
+    @motivating
+    Scenario Outline: Default menu item state in the side navigation bar
+        Given a user looks at the <menuItem> menu item in the side navigation bar
+        Then they see that the background color is <backgroundColor>
+        And text color is <textColor>
+        And text type is  <textType>
+        And the icon color is <iconColor>
+        And the dropdown line is <dropdownLine>
+        And the arrow points <arrowDirection>
+
+        Examples:
+
+            | <menuItem>                                  | <backgroundColor> | <textColor>         | <textType> | <iconColor>         | <dropdownLine>      | <arrowDirection> |
+            | Core Category Section                       | FFFFFF            | 003B45 (dark green) | normal     | 003B45 (dark green) | FFFFFF              | N/A              |
+            | Core Category Section with navigation arrow | FFFFFF            | 003B45 (dark green) | normal     | 003B45 (dark green) | FFFFFF              | to the right     |
+            | Parent                                      | FFFFFF            | 003B45 (dark green) | normal     | 003B45 (dark green) | FFFFFF              | N/A              |
+            | Parent with dropdown arrow                  | FFFFFF            | 003B45 (dark green) | normal     | 003B45 (dark green) | FFFFFF              | down             |
+            | Child                                       | FFFFFF            | 003B45 (dark green) | normal     | 003B45 (dark green) | 003B45 (dark green) | N/A              |
+            | Child with dropdown arrow                   | FFFFFF            | 003B45 (dark green) | normal     | 003B45 (dark green) | 003B45 (dark green) | down             |
+
```

Comment:
It would be great to have a general UI spec on what the first level of the nav looks like/does